src/mscorlib/src/Internal/Runtime/Augments/AppContextAugments.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Reflection;
+
+namespace Internal.Runtime.Augments
+{
+    /// <summary>For internal use only.  Exposes runtime functionality to the Appdomain implementation in corefx.</summary>
+    public static class AppContextAugments
```

Comment:
Yes I do have it there in System.AppContext in corefx PR https://github.com/dotnet/corefx/pull/11275 . This is exposing runtime implementation to corefx. This isn't public surface.